corehq/util/urlsanitize/test/test_urlsanitize.py

```diff
@@ -0,0 +1,54 @@
+import ipaddress
+import socket
+
+from ..urlsanitize import PossibleSSRFAttempt, sanitize_user_input_url, CannotResolveHost, InvalidURL
+
+RAISE = object()
+RETURN = object()
+
+GOOGLE_IP = ipaddress.ip_address(socket.gethostbyname('google.com'))
```

Comment:
This would ideally be resolved in test setup, not at module import time since it hits the network and could fail. Maybe move it into a `setup_module` function?